user_guide.md

```diff
@@ -32,7 +32,67 @@ func main() {
 Cobra provides its own program that will create your application and add any
 commands you want. It's the easiest way to incorporate Cobra into your application.
 
-[Here](https://github.com/spf13/cobra/blob/master/cobra/README.md) you can find more information about it.
+Install the cobra generator with the command `go install github.com/spf13/cobra/cobra`. 
```

Comment:
The documentation of the CLI itself is located in `cobra/README.md`. Hence, this feels duplicated and now not matching. Was your purpose to actually remove `cobra/README.md`? Maybe this content should be written there?

Reply:
Oh, I totally missed that this was where it was. I was surprised their wasn't more documentation. I only noticed the /README.md and this user_guide.md file. This is especially embarrassing because I'm pretty sure I wrote it originally. I'll merge the updated docs into this one and add links to make this more clear again.

Reply:
Addressed in b4ab6634be8a507c3047efdaa89a789d3b6504c0